index.html

```diff
@@ -1472,6 +1497,15 @@ <h3>5.2 Challenges</h3>
           implemented. The result is a lightning fast framework.
         </p>
 
+        <h4>5.2.3 Domain</h4>
+
+        <p>
+          Serverless orchestration, and the subsets of orchestration with AWS
+          Lambdas and AWS Step Functions is a very niche topic. Because of this,
+          we didn't really have any frameworks or solutions in this space to
+          reference, leaving us on our own to blaze our own trail.
+        </p>
```

Comment:
Similar to before, there are some misplaced commas and a fleeing from 3rd person with use of `we` in the first person plural. On a whole, the paragraph was rephrased to express its intent more clearly without using absolutes.

          Serverless orchestration is a very niche topic. Although AWS Lambda
          is a popular service, orchestration is often done company by company
          without adhering to a common set of shared best practices. Navigating
          common approaches required extensive, in depth inquiry. Further, AWS
          Step Functions offers many benefits but is not broadly used; as a result,
          few frameworks or solutions exist to provide insight into development.
          Therefore, the Maestro team had to be quite innovative to provide a
          framework in this space.